ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureCloudDestination.java

```diff
@@ -376,6 +376,27 @@ public boolean updateBlobExpiration(BlobId blobId, long expirationTime) throws C
     }
   }
 
+  /**
+   * Filter out {@link CloudBlobMetadata} objects from lastUpdateTime ordered {@code cloudBlobMetadataList} whose
+   * lastUpdateTime is {@code lastUpdateTime} and id is in {@code lastReadBlobIds}.
+   * @param cloudBlobMetadataList list of {@link CloudBlobMetadata} objects to filter out from.
```

Comment:
Remove "from"

Reply:
Actually I think from is needed. "CloudBlobMetadatalist" is a list of objects to filter out "from". If we remove "from" it would mean these are the objects to be filtered out.

Reply:
Ignore this. I misunderstood. will fix it.